syz-ci/manager.go

```diff
@@ -875,8 +880,9 @@ func (mgr *Manager) uploadCoverReport() error {
 	return nil
 }
 
-func (mgr *Manager) uploadCoverStat(fuzzingMinutes int) error {
-	if !mgr.managercfg.Cover || mgr.cfg.CoverPipelinePath == "" {
+func (mgr *Manager) uploadCoverJSONLToGCS(mgrSrc, gcsDest string, f func(io.Writer, *json.Decoder, time.Time) error,
```

Comment:
Passing the non-changing `time.Time` value into the callback looks excessive. If you absolutely want to fix time timestamp, it may be better to pass it as a `uploadCoverJSONLToGCS` argument.

Reply:
I want all the records in one jsonl file to be the same. How will the time move to uploadCoverJSONLToGCS help here?

Reply:
Minus one parameter from the callback.

Reply:
I mean imagine we moved the timestamp out from the callback signature.
Where can I get the timestamp for every generated json object?

Reply:
Got it. I'll change.